src/main/java/com/liferay/blade/cli/CreateCommand.java

```diff
@@ -73,7 +73,7 @@ else if (!_isExistingTemplate(template)) {
 		if (_args.getDir() != null) {
 			dir = new File(_args.getDir().getAbsolutePath());
 		}
-		else if (template.equals("theme") || template.equals("layout-template") ||
+		else if (template.startsWith("war") || template.equals("theme") || template.equals("layout-template") ||
```

Comment:
Hey @christopherbryanboyd  can you send another commit that includes an integration test?  thanks!

Reply:
Hey @gamerson , tests have now been added that test both "war-hook" and "war-mvc-portlet" project types, ensuring they now go into the "wars" folder.